hack/code/instancetype_testdata_gen/main.go

```diff
@@ -147,13 +147,14 @@ func getInstanceTypeInfo(info *ec2.InstanceTypeInfo) string {
 		fmt.Fprintf(src, "NvmeSupport: aws.String(\"%s\"),\n", lo.FromPtr(info.EbsInfo.NvmeSupport))
 		fmt.Fprintf(src, "},\n")
 	}
-	if info.InferenceAcceleratorInfo != nil {
-		fmt.Fprintf(src, "InferenceAcceleratorInfo: &ec2.InferenceAcceleratorInfo{\n")
-		fmt.Fprintf(src, "Accelerators: []*ec2.InferenceDeviceInfo{\n")
-		for _, elem := range info.InferenceAcceleratorInfo.Accelerators {
-			fmt.Fprintf(src, getInferenceAcceleratorDeviceInfo(elem))
+	if info.NeuronInfo != nil {
+		fmt.Fprintf(src, "NeuronInfo: &ec2.NeuronInfo{\n")
+		fmt.Fprintf(src, "NeuronDevices: []*ec2.NeuronDeviceInfo{\n")
+		for _, elem := range info.NeuronInfo.NeuronDevices {
+			fmt.Fprintf(src, getNeuronDeviceInfo(elem))
 		}
 		fmt.Fprintf(src, "},\n")
+		fmt.Fprintf(src, "TotalNeuronDeviceMemoryInMiB: aws.Int64(%d),\n", lo.FromPtr(info.NeuronInfo.TotalNeuronDeviceMemoryInMiB))
```

Comment:
We should drop this if we don't need `LabelInstanceAcceleratorMemory`?

Reply:
removed in d4eba39c9143d9903112cf0d42e65fd2a2b0bafc